### PR TITLE
Feature: custom master chooser

### PR DESCRIPTION
**Pull request by jubeira**
This PR modifies `RosActivity` to be able to launch a custom `MasterChooser` instead of the standard one.
This solves #258 ; see the issue for the motivation for this change.

Any feedback about this is welcome.
@stratomda @adamantivm FYI.

### CODE REVIEW

File: android_10/src/org/ros/android/RosActivity.java [hunk 1]
```diff
@@ -98,17 +148,44 @@ public NodeMainExecutorServiceListener getServiceListener()
...
   protected RosActivity(String notificationTicker, String notificationTitle) {
     this(notificationTicker, notificationTitle, null);
   }
 
+  /**
+   * Custom Master URI constructor.
+   * Use this constructor to skip launching {@link MasterChooser}.
+   * @param notificationTicker Title to use in Ticker notifications.
+   * @param notificationTitle Title to use in notifications.
+   * @param customMasterUri URI where to connect to.
```
Comment (adamantivm) [inline android_10/src/org/ros/android/RosActivity.java]: wording: 'URI of the ROS master to connect to'

File: android_10/src/org/ros/android/RosActivity.java [hunk 2]
```diff
@@ -98,17 +148,44 @@ public NodeMainExecutorServiceListener getServiceListener()
...
     nodeMainExecutorServiceConnection = new NodeMainExecutorServiceConnection(customMasterUri);
   }
 
+  /**
+   * Custom MasterChooser constructor.
+   * Use this constructor to specify which {@link Activity} should be started in place of {@link MasterChooser}.
+   * @param notificationTicker Title to use in Ticker notifications.
+   * @param notificationTitle Title to use in notifications.
+   * @param activity {@link Activity} to launch instead of {@link MasterChooser}.
+   * @param requestCode Code to be sent to the specified {@link Activity}.
```
Comment (adamantivm) [inline android_10/src/org/ros/android/RosActivity.java]: maybe expand a tiny bit ... code to be send to the given activity at what time? for what?

**Review by adamantivm — APPROVED** [review]
nice work! I only added a few minor doc nits.

**Comment by jubeira** [comment]
Thanks for your review @adamantivm , nits were addressed.

**Pull request merged**